site/gatsby-site/src/components/checklists/ChecklistsIndex.js

```diff
@@ -14,28 +15,102 @@ import {
   statusIcon,
   statusColor,
   generateId,
+  abbreviatedTag,
 } from 'utils/checklists';
 import { FIND_CHECKLISTS, INSERT_CHECKLIST, DELETE_CHECKLIST } from '../../graphql/checklists';
 import useToastContext, { SEVERITY } from '../../hooks/useToast';
 
 const ChecklistsIndex = ({ users }) => {
   const { t } = useTranslation();
 
+  const addToast = useToastContext();
+
   const { user } = useUserContext();
 
-  const addToast = useToastContext();
+  const loggedIn = user?.providerType != 'anon-user';
 
   const [insertChecklist] = useMutation(INSERT_CHECKLIST);
 
-  const { data: checklistsData, loading: checklistsLoading } = useQuery(FIND_CHECKLISTS);
+  /************************** Get Checklists **************************/
+  const { data: checklistsData, loading: checklistsLoading } = useQuery(FIND_CHECKLISTS, {
+    variables: { query: { owner_id: user?.id } },
+    skip: !user?.id,
+  });
 
+  // In useState so that on deleting a checklist,
+  // we can remove it from display immediately.
   const [checklists, setChecklists] = useState([]);
 
   useEffect(() => {
     setChecklists(checklistsData?.checklists || []);
   }, [checklistsData]);
 
-  const loggedIn = user.providerType != 'anon-user';
+  const riskQueryableChecklists = (checklistsData?.checklists || []).filter(
+    (c) => allTags(c).length > 0
+  );
+
+  /***************************** Get Risks ****************************/
+  const skipRisksQuery = !riskQueryableChecklists.length > 0;
+
+  // We want to query the risks for each checklist.
+  // To do that, we construct multiple labeled risk queries:
+  //
+  // Example
+  // -------------------------------------------------------------------
+  // query {
+  //   GMFKnownAITechnologyTransformer:
+  //   risks(input: {tags: ["GMF:Known AI Technology:Transformer"]}) {
+  //     tags
+  //     title
+  //   }
+  //   GMFKnownAITechnicalFailureUnderspecification:
+  //   risks(input: {tags: ["GMF:Known AI Technical Failure:Underspecification"]}) {
+  //     tags
+  //     title
+  //   }
+  // }
+  const riskQuery = skipRisksQuery
+    ? 'query { thisWontRunBut { useQueryMakesUseProvideValidGraphQL } }'
```

Comment:
```suggestion
    ? 'query { thisWontRunBut { useQueryMakesUsProvideValidGraphQL } }'
```
Please consider an inline comment above this.

Help me understand: why use a conditional here and create a mock/dud query, and not conditionally skip the `useQuery` hook below, or rely on `{ skip: skipRisksQuery }` ?

Reply:
React hooks can't run conditionally. We are relying on `{ skip: skipRisksQuery }`, but it will still throw an exception if first parameter isn't graphql.